src/App.vue

```diff
@@ -52,7 +52,7 @@ export default {
     MilestoneSelector,
     Milestone,
     Release,
-    AfterDeployChecklist,
+    AfterDeployChecklist
```

Comment:
The trailing commas are applied quite consistently, so you could enforce trailing commas in `.prettierrc` if you wanted to keep them.

https://prettier.io/docs/en/options.html#trailing-commas

Reply:
Added 👍